### ISSUE TITLE
Mockito dependency needs to be test scoped

### ISSUE THREAD

**Issue by wonmo01** [confluentinc/parallel-consumer#375]
From parent pom.xml, mockito dependency is not scoped as test.
https://github.com/confluentinc/parallel-consumer/blob/master/pom.xml#L335-L339
```
        <dependency>
            <groupId>org.mockito</groupId>
            <artifactId>mockito-junit-jupiter</artifactId>
            <version>${mockito.version}</version>
<!--            <scope>test</scope> <<<<<<<<<<<< MISSING --> 
        </dependency>
```

**Comment by wonmo01** [comment]
@astubbs I am not sure if this is intentional, but is there any chance this can be changed to test scoped?

**Comment by astubbs** [comment]
Yes, that needs fixing

**Comment by astubbs** [comment]
Released as 0.5.2.2
